### DISCUSSION TITLE
LTMEAT funny haha

### DISCUSSION THREAD

**Discussion by 3s7vn** [3kh0/ext-remover#649] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
im angrey!11!!

**Comment by pazza2234** [comment]
Wait I have not been here for a while but you people don't know? it simple. go to Chrome extensions, click on the details button. of the extension you want to disable, and get the extension id from the URL. go to chrome-extension://[exstention ID]/manifest.json and make a bookmark there for next time. create a bookmark and make it go to chrome://kill, and in another bookmark make it chrome://hang. click chrome://kill, then spam click chrome://hang, and while still spamming it click reload. from when you click on the chrome://kill bookmark you got about 3 seconds to do the chrome://hang + Relode step.

**Reply by pazza2234** [reply]
also, there is a wifi method where you got more time. click on kill,  then click on the bookmark back to the extension manifest, and click hang. then go to a webpage with out enabling wifi. enable wifi, and let the page auto reload. then it should work. that is the method my friends with slow fingers and low IQ use.

**Reply by pazza2234** [reply]
All I had to do is read the read me. It already exists why am I talking about someone ban me.

**Comment by 3kh0** [comment]
wild

**Reply by RavenChill-Dev** [reply]
Ik how to do it without bookamrlets, see my help

**Reply by 3s7vn** [reply]
where do i check your help?

**Reply by RavenChill-Dev** [reply]
in diccusion on this https://github.com/3kh0/ext-remover/discussions/ my name should show up at the list, and my ltmeat not working

**Reply by 3s7vn** [reply]
ok thanks

**Comment by 3s7vn** [comment]
GUYS WE GOT MORE YES THAN NO!1!!
